### PR TITLE
fix bugs in generating wsj0-mix dataset with wv1 && loading model acc…

### PR DESCRIPTION
**Pull request by hangtingchen**
fix bugs in generating wsj0-mix dataset with wv1 instead of wv2 && loading model according to the epoch number

### CODE REVIEW

**Comment by mpariente** [comment]
Thanks a lot for the fix !
Next time, please make two separated PRs as those are two different problems, it makes easier to track changes afterwards.

File: egs/wsj0-mix/DeepClustering/model.py [hunk 1]
```diff
@@ -153,8 +153,11 @@ def load_best_model(train_conf, exp_dir):
     except FileNotFoundError:
         # Get last checkpoint
         all_ckpt = os.listdir(os.path.join(exp_dir, 'checkpoints/'))
-        all_ckpt.sort()
-        best_model_path = os.path.join(exp_dir, 'checkpoints', all_ckpt[-1])
+        all_ckpt = [(ckpt, int("".join(filter(str.isdigit, \
```
Comment (mpariente) [inline egs/wsj0-mix/DeepClustering/model.py:156]: You don't need the backslash.

File: egs/wsj0-mix/DeepClustering/model.py [hunk 2]
```diff
@@ -153,8 +153,11 @@ def load_best_model(train_conf, exp_dir):
...
+                    os.path.basename(ckpt))))) \
```
Comment (mpariente) [inline egs/wsj0-mix/DeepClustering/model.py:157]: This should be aligned with the `filter` starting parenthesis.

**Comment by mpariente** [comment]
This will solve #164 and also #165 and the use of wv2 explained the poor performance in #138.

**Comment by hangtingchen** [comment]
> This will solve #164 and also #165 and the use of wv2 explained the poor performance in #138.

We have updated the code.

**Comment by mpariente** [comment]
Thanks ! 
The formatting is still not ok and I think you disabled "edit from the maintainers", I'll merge and fix it.

**Comment by hangtingchen** [comment]
Thanks. Any guide for the coding style?

**Comment by mpariente** [comment]
https://github.com/psf/black

**Pull request merged**